plugins/workbench/src/index.ts

```diff
@@ -26,6 +26,7 @@ import { plugin } from '@anticrm/platform'
 export interface Application extends Doc {
   label: IntlString
   icon: Asset
+  hidden: boolean
```

Comment:
I would prefer it to be `hidden?: boolean` so if it is defined, it will be used.

Reply:
I too, but we don't have $ne, and I don't want to filter this on front